### PR TITLE
Claimed books component created and Nav bar adresses changed

### PR DESCRIPTION
**Pull request by Andrei-Trifu**
1.Claimed books component created
2.Nav bar links now functional
3.Claimed books page styled so it looks similar to the Allbooks page

### CODE REVIEW

File: src/Components/Nav/Nav.jsx [hunk 1]
```diff
@@ -3,14 +3,14 @@ import "./Nav.css";
 function Nav() {
```
Comment (Kel-E-creator) [inline src/Components/Nav/Nav.jsx:4]: Please add white space between lines 2 & 3 :-)

File: src/Components/Nav/Nav.jsx [hunk 2]
```diff
@@ -3,14 +3,14 @@ import "./Nav.css";
...
   return (
     <nav className="navigation">
```
Comment (Kel-E-creator) [inline src/Components/Nav/Nav.jsx:6]: Please add white space between lines 4 & 5

File: src/Components/Nav/Nav.jsx [hunk 3]
```diff
@@ -3,14 +3,14 @@ import "./Nav.css";
...
         <h1>Book Swap</h1>
-      </div>
+      </Link>
       <div className="links">
-        <NavLink activeClass="active" to="availableBooks">
+        <NavLink activeClass="active" to="/">
           Available books
         </NavLink>
-        <NavLink activeClass="active" to="claimedBooks">
+        <NavLink activeClass="active" to="/books/claimed">
```
Comment (Kel-E-creator) [inline src/Components/Nav/Nav.jsx:14]: Please remove semi-colon after export default Nav :-)

File: src/Components/HomePage/ClaimedBooks/ClaimedBooks.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+@media screen and (max-width: 850px) {
+  .claimed-books {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+}
+@media screen and (max-width: 850px) {
```
Comment (Kel-E-creator) [inline src/Components/HomePage/ClaimedBooks/ClaimedBooks.css:16]: add whitespace between lines 14 and 15 :-)

File: src/Components/HomePage/ClaimedBooks/ClaimedBooks.jsx
Comment (Kel-E-creator) [inline src/Components/HomePage/ClaimedBooks/ClaimedBooks.jsx:1]: remove semi colons from the end of the lines in the jsx file 👍

File: src/Components/HomePage/ClaimedBooks/ClaimedBooks.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  const [ClaimedBooks, setClaimedBooks] = useState([]);
+
+  useEffect(function () {
+    fetch("https://book-swap-api.dev.io-academy.uk/api/books?claimed=1")
+      .then(function (response) {
+        return response.json();
+      })
+      .then(function (bookData) {
+        setClaimedBooks(bookData.data);
+        console.log(bookData.data);
```
Comment (Kel-E-creator) [inline src/Components/HomePage/ClaimedBooks/ClaimedBooks.jsx]: remove console.log on line 15 pls 👍

**Review by Kel-E-creator — CHANGES_REQUESTED** [review]
please see our comments

File: src/Components/HomePage/ClaimedBooks/ClaimedBooks.css (hunk 4 above)
Comment (Andrei-Trifu) [reply src/Components/HomePage/ClaimedBooks/ClaimedBooks.css:16]: done

File: src/Components/HomePage/ClaimedBooks/ClaimedBooks.jsx
Comment (Andrei-Trifu) [reply src/Components/HomePage/ClaimedBooks/ClaimedBooks.jsx:1]: I tried but my VsCode extension is not letting me do it. Sorry

File: src/Components/HomePage/ClaimedBooks/ClaimedBooks.jsx (hunk 5 above)
Comment (Andrei-Trifu) [reply src/Components/HomePage/ClaimedBooks/ClaimedBooks.jsx]: done

File: src/Components/Nav/Nav.jsx (hunk 1 above)
Comment (Andrei-Trifu) [reply src/Components/Nav/Nav.jsx:4]: done

File: src/Components/Nav/Nav.jsx (hunk 2 above)
Comment (Andrei-Trifu) [reply src/Components/Nav/Nav.jsx:6]: again VsCode thinks it's better this way, and I don't see why either, it already has an indentation

File: src/Components/Nav/Nav.jsx (hunk 3 above)
Comment (Andrei-Trifu) [reply src/Components/Nav/Nav.jsx:14]: same as before :) vscode

**Review by Kel-E-creator — APPROVED** [review]

**Pull request merged**